projects/com.oracle.truffle.llvm.parser.factories/src/com/oracle/truffle/llvm/parser/factories/LLVMRuntimeIntrinsicFactory.java

```diff
@@ -118,6 +119,7 @@
         intrinsics.put("@truffle_import", LLVMTruffleImportFactory.getInstance());
         intrinsics.put("@truffle_import_cached", LLVMTruffleImportCachedFactory.getInstance());
         intrinsics.put("@truffle_address_to_function", LLVMTruffleAddressToFunctionFactory.getInstance());
+        intrinsics.put("@truffle_get_arg", LLVMTruffleGetArgFactory.getInstance());
```

Comment:
Or we could optionally pass caller frames to the callee, like we do in Ruby sometimes.